lib/assets/constants.dart

```diff
@@ -0,0 +1,24 @@
+// App
+const double APP_BAR_HEIGHT = 107.0;
+const String APP_TITLE = "Medium app screen layout";
+
+// Images
+const int NETWORK_IMAGES_MAX_ID = 1084;
+const String NETWORK_IMAGES_URL = "https://picsum.photos";
+
+// Fake stories
+const String FAKE_STORY_TITLE = 'The Story We Tell Ourselves';
+const String FAKE_STORY_DESCRIPTION = 'How do you see luck?';
+const String FAKE_STORY_AUTHOR = 'Darren Matthews';
+const String FAKE_STORY_TIME_DETAILS = '6 hours ago  -  2 min read  ';
+
+// Card
+const int CARD_IMAGE_WIDTH = 350;
```

Comment:
Just remember that if your constant is used only in a single place (single widget for instance), it's better to declare it there instead of separate file (to avoid jumping from file to file).